setup.cfg

```diff
@@ -3,5 +3,18 @@ max-line-length = 119
 
 [flake8]
 max-line-length = 119
-ignore = W503
+ignore = W503,W504
```

Comment:
I have fixes for some of the ignored issues.

Reply:
I'll remove this part.